printer_mods/README.md

```diff
@@ -254,6 +254,7 @@ like so:
 | Mellow | [V0 Fly-Gemini](./Mellow/V0_Fly-Gemini) | Fly-Gemini adapted model for V0 | :heavy_check_mark: | :x: | :x: | :x: | :x:|
 | mikepthomas | [MKS Mosfet Mount](./mikepthomas/MKS_Mosfet_Mount) | MKS Mosfet Mount based of the Trident Raspberry Pi Mount. | :x: | :heavy_check_mark: | :heavy_check_mark: | :x: | :heavy_check_mark: |
 |MikesMods|[LGX mount for V0.1 with 2nd X rail](./MikesMods/LGX_mount_for_V0.1_with_2nd_X_rail) |V0.1 mod for mounting Bondtech LGX extruder with addition of 2nd X rail on the back of the 1515 extrusion. | :heavy_check_mark: | :x: | :x: |:x:|:x:|
+| Minsekt | [Rear Umbilical](https://github.com/Minsekt/VoronUsers/tree/master/printer_mods/Minsekt/2.4/Rear_Umbilical) | Rear Umbilical mounting solution | :x: | :heavy_check_mark: | :x: | :x: | :x: |
```

Comment:
you missed one of the links to change to relative links here.

Reply:
Also missed one in the readme file too

Reply:
fixed, changed all to relative links and updated name